src/bipartite.jl

```diff
@@ -1,24 +1,25 @@
 """
     BipartiteView{T<:Real} <: AbstractGraph{Int64}
 
-Create a bipartite view of a hypergraph `h`.
+Represents a bipartite view of a hypergraph `h`.
 Note this is a view - changes to the original hypergraph will be automatically reflected in the view.
 
+**Constructors**
+
+BipartiteView(h::Hypergraph{T}) where {T<:Real}
+
 The bipartite view of a hypergraph is suitable for processing with the LightGraphs.jl package.
 Several LightGraphs methods are provided for the compability.
 
 """
 struct BipartiteView{T<:Real} <: AbstractGraph{Int}
     h::Hypergraph{T}
-    function BipartiteView{T}(h::Hypergraph{T}) where {T<:Real}
-        new(h)
-    end
 end
 
 
 LightGraphs.nv(b::BipartiteView) = length(b.h.v2he)+length(b.h.he2v)
 
-LightGraphs.vertices(b::BipartiteView) = Base.OneTo(nv(b))
+LightGraphs.vertices(b::BipartiteView) = Base.OneTo(LightGraphs.nv(b))
```

Comment:
why it is required to qualify it here?

Reply:
Then you should not do `using LightGraphs` but rather `import LightGraphs` in src/SimpleHypergraphs.jl.